tests/phpunit/tests/date/wpDate.php

```diff
@@ -61,4 +61,55 @@ public function test_should_keep_localized_slashes() {
 
 		$this->assertSame( $string, wp_date( 'F', $datetime->getTimestamp(), $utc ) );
 	}
+
+	/**
+	 * Test if no timestamp provides
+	 *
+	 * @ticket 53485
+	 */
+	public function test_no_timestamp() {
+
+		$this->assertSame( (string) strtotime( 'now' ), wp_date( 'U' ) );
+	}
+
+	/**
+	 * Test if format is set to F weekday_abbrev
+	 *
+	 * @ticket 53485
+	 */
+	public function test_format_F() {
+		$utc      = new DateTimeZone( 'UTC' );
+		$datetime = new DateTimeImmutable( '2019-10-17', $utc );
+
+		$this->assertSame( 'Thu', wp_date( 'D', $datetime->getTimestamp(), $utc ) );
+	}
+
+	/**
+	 * Test if format is set to M month_abbrev
+	 *
+	 * @ticket 53485
+	 */
+	public function test_format_M() {
+		$utc      = new DateTimeZone( 'UTC' );
+		$datetime = new DateTimeImmutable( '2019-10-17', $utc );
+
+		$this->assertSame( 'Oct', wp_date( 'M', $datetime->getTimestamp(), $utc ) );
+	}
+
+	/**
+	 * Test if format is set to M month_abbrev
+	 *
+	 * @ticket 53485
+	 */
+	public function test_wp_date_filter() {
```

Comment:
Hm hm, do we need an explicit test that filter works for a specific function? Is there a precedent in tests for doing this every time?

Reply:
yes this makes sure the filter is there